### PR TITLE
Provide the initial migration SN-385

### PR DESCRIPTION
**Pull request by vasiliy-chefonov**
https://saritasa.atlassian.net/browse/SN-385

### CODE REVIEW

File: demo/Saritasa.NetForge.Demo.Net7/Infrastructure/Startup/DatabaseInitializer.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿using Extensions.Hosting.AsyncInitialization;
+using Microsoft.EntityFrameworkCore;
+
+namespace Saritasa.NetForge.Demo.Net7.Infrastructure;
```
Comment (AlexanderMakarov124) [inline demo/Saritasa.NetForge.Demo.Net7/Infrastructure/Startup/DatabaseInitializer.cs:4]: Should we adjust namespace to `Saritasa.NetForge.Demo.Net7.Infrastructure.Startup`?

File: demo/Saritasa.NetForge.Demo.Net7/Infrastructure/Startup/HealthCheck/HealthCheckOptionsSetup.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    /// <summary>
+    /// Returns <see cref="HealthCheckOptions" />.
+    /// </summary>
+    public HealthCheckOptions Setup(HealthCheckOptions options)
+    {
+        options.ResponseWriter = WriteResponse;
+        return options;
+    }
+
+    private static async Task WriteResponse(HttpContext context, HealthReport report)
```
Comment (AlexanderMakarov124) [inline demo/Saritasa.NetForge.Demo.Net7/Infrastructure/Startup/HealthCheck/HealthCheckOptionsSetup.cs:29]: Do we need `Async` suffix here?

File: demo/Saritasa.NetForge.Demo.Net7/Models/Address.cs [hunk 3]
```diff
@@ -16,22 +18,22 @@ public class Address
     /// <summary>
     /// The street name and number.
     /// </summary>
-    public string Street { get; set; }
+    public string Street { get; set; } = string.Empty;
```
Comment (AlexanderMakarov124) [inline demo/Saritasa.NetForge.Demo.Net7/Models/Address.cs:21]: Maybe we can use `required` modifier instead of `string.Empty`?

**Review by AlexanderMakarov124 — APPROVED** [review]

**Review by ko-vasilev — APPROVED** [review]

File: docker-compose.yaml
Comment (ko-vasilev) [inline docker-compose.yaml:1]: Should it be put in the `demo` folder?

**Pull request merged**